pipelines/build/common/build_base_file.groovy

```diff
@@ -1034,6 +1034,21 @@ class Builder implements Serializable {
             if (publish || release) {
                 if (release) {
                     context.println 'NOT PUBLISHING RELEASE AUTOMATICALLY, PLEASE SEE THE RERUN RELEASE PUBLISH BINARIES LINKS'
+                    if (context.HUDSON_URL.contains('adoptium')) {
```

Comment:
I'm guessing HUDSON_URL works because it is already being used [here](https://github.com/adoptium/ci-jenkins-pipelines/blame/36b02deb065cb948706342f76107b63df7f91f36/pipelines/build/common/build_base_file.groovy#L791). But I would think we should probably use JENKINS_URL. I'm not sure if HUDSON_URL will go away someday.